### PR TITLE
Update Serving Reviewers and drop in-active reviewers/members

### PR DESCRIPTION
**Pull request by dprotaso**

### CODE REVIEW

**Comment by dprotaso** [comment]
/hold

I'm dropping teams that are referenced here - https://github.com/knative/pkg/pull/2672

File: peribolos/knative-OWNERS_ALIASES [hunk 1]
```diff
@@ -89,94 +84,53 @@ aliases:
...
   security-wg-leads:
   - evankanderson
   security-writers:
   - evankanderson
-  serving-observability-reviewers:
-  - skonto
-  - yanweiguo
-  serving-observability-writers:
-  - yanweiguo
+  serving-approvers:
```
Comment (psschwei) [inline peribolos/knative-OWNERS_ALIASES:117]: what's the difference between serving-approvers and serving-writers?

File: peribolos/knative-OWNERS_ALIASES [hunk 2]
```diff
@@ -89,94 +84,53 @@ aliases:
...
+  - nak3
```
Comment (dprotaso) [inline peribolos/knative-OWNERS_ALIASES:118]: FYI: @nak3 was a Networking Lead who stepped down recently.  https://github.com/knative/community/pull/1191

Given Serving absorbed the Networking WG I have no issue with him retaining his approver role. @nak3 if you'd rather not be here let me know.

File: peribolos/knative-OWNERS_ALIASES (hunk 1 above)
Comment (dprotaso) [reply peribolos/knative-OWNERS_ALIASES:117]: writers includes approvers and wg leads

**Review by psschwei — COMMENTED** [review]
/lgtm

**Pull request merged**